deployments/fuji/usdc/configuration.json

```diff
@@ -29,15 +29,15 @@
       "borrowCF": 0.7,
       "liquidateCF": 0.75,
       "liquidationFactor": 0.93,
-      "supplyCap": "35000"
+      "supplyCap": "35000e8"
     },
     "WAVAX": {
       "priceFeed": "0x5498BB86BC934c8D34FDA08E81D444153d0D06aD",
       "decimals": "18",
       "borrowCF": 0.82,
       "liquidateCF": 0.85,
       "liquidationFactor": 0.93,
-      "supplyCap": "1000000"
+      "supplyCap": "1000000e18"
```

Comment:
We're sure we prefer this being redundant? In that case, maybe worth going so far as removing `_` too so can write `1_000_000e18`

Reply:
Good point, added support for underscores.

Reply:
Honestly, I can be swayed both ways. I'm starting to appreciate the simplicity of the other approach. We just have to make sure to apply the right scales to some other values like `baseMinForRewards`

Reply:
Yeah, up to you if you want to merge, I often think less is more though

Reply:
I'll merge this since it reduces the amount of configuration diffs we have against `main`. Functionally, both approaches are the same so we could revisit in a separate PR that focuses on refactoring the configuration.